doc/reference/configuration/configuration_reference.rst

```diff
@@ -730,8 +730,46 @@ This section describes options related to providing connection settings to a :re
     | Default: 3
     | Environment variable: TT_CONFIG_STORAGE_TIMEOUT
 
+..  _configuration_reference_console:
 
+console
+-------
+
+Tarantool output can be written to a file or Unix socket.
+
+..  NOTE::
+
+    ``console`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+* :ref:`console.enabled <config_console_enabled>`
+* :ref:`console.socket <config_console_socket>`
 
+.. _config_console_enabled:
+
+.. confval:: console.enabled
+
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Whether to enable Tarantool output to a file or Unix socket.
```

Comment:
Whether to listen on the Unix socket provided in the `console.socket` option. If the option is set to `false`, the admin console is disabled.

See the comment above for details.